addon/components/power-select.js

```diff
@@ -188,6 +188,19 @@ export default Component.extend({
       && (!this.get('search') || publicAPI.lastSearchedText.length > 0);
   }),
 
+  ariaAutocomplete: computed('searchEnabled', function() {
+    return this.get('searchEnabled') ? 'list' : 'none';
+  }),
+
+  highlightedId: computed('publicAPI.highlighted', function() {
+    let { highlighted, results, uniqueId } = this.get('publicAPI');
+    return `${uniqueId}-${indexOfOption(results, highlighted)}`;
+  }),
+
+  resultsCountId: computed('publicAPI.uniqueId', function() {
+    return `ember-power-select-results-count-${this.get('publicAPI.uniqueId')}`;
+  }),
+
```

Comment:
This PR is great!  My only real comment would be wrt to the computed cache breaking keys.  For example, I wonder if `searchEnabled` or `publicAPI.highlighted` or `publicAPI.uniqueId` was removed if your tests would still pass?